packages/terra-data-grid/src/DataGrid.jsx

```diff
@@ -178,6 +178,9 @@ const DataGrid = injectIntl((props) => {
   const tableContainerRef = useRef();
   const handleFocus = useRef(true);
 
+  const lastActiveRowId = useRef();
+  const lastActiveColumnId = useRef();
```

Comment:
Since the is a reference variable, I think you can just create one single object with the two properties.  Also, I still think we should use "focused" instead of "active".  They have two different meanings in CSS.  It would also be easier to follow if the naming conventions were the same.